src/views/catalog/CreateFromInstanceTypes/utils/utils.ts

```diff
@@ -98,7 +86,7 @@ export const generateVM = (
       instancetype: {
         // inferFromVolume: `${virtualmachineName}-disk`,
```

Comment:
I know you weren't changing this line, but don't you know the reason of this comment? Feel unnecessary here...

Reply:
it's not sure that it will be ever useful, so in case it does I will add it again :)